pkg/network/usm/utils/file_registry.go

```diff
@@ -113,6 +120,18 @@ var (
 	ErrPathIsAlreadyRegistered = errors.New("path is already registered")
 )
 
+// getSamplePath returns a sample path for a given path without the prefix /usr.
+func getSamplePath(path string) string {
+	index := strings.Index(path, "/usr")
+	samplePath := path
+
+	if index != -1 {
+		samplePath = path[index+len("/usr"):]
+	}
+
+	return samplePath
+}
```

Comment:
what? why do you remove the `/usr` if it exists?
and you have better methods to do so (see strings.TrimPrefix)

Reply:
I used this function because I was trying to find it not only as a prefix but also in the middle of the path. I have changed it to use only `/proc` as the prefix, so I have updated the function to use TrimPrefix accordingly.